python/py-scipy/Portfile

```diff
@@ -84,6 +84,15 @@ if {${name} ne ${subport}} {
         compilers.allow_arguments_mismatch yes
 
     } else {
+        # latest upstream version, supports Python 3.7+
```

Comment:
From the main info here: `it now uses fetch.type git and calls git submodule to get the vendored boost dependency.` ... let me do some testing & see what's what here. I really don't like having to go this route unless it's absolutely required to do so.

Reply:
Sure - feel free! I am not a big fan either and would prefer to use the MP version of boost. But I couldn't find a very easy way of specifying it and admittedly didn't attempt to start patching stuff. Also, it might be wise to stick with the vendored version as the SciPy folks have tested this.

Reply:
Yes sticking with a specific Boost is often the best thing to do ... hopefully the new Boost MP options will be good enough for what SciPy needs

Reply:
the latest commit in the SciPy repository regarding this is :`REV: revert to Boost 1.75 to avoid constexpr bug with gcc <7.1 `; so I guess we don't have a `boost` version in MP that is compatible with it. 

I did consider switching to PyPI as `master_sites` since the files there presumably contain the vendored `boost` library. One would probably need to switch then to PyPI for the older versions of SciPy as well. I think that should work, but haven't tested it.

Reply:
Using MP boost will not be difficult. For this release, the submodule is 1.76.0 ... regardless of the GCC issue as you note. It's straight forward to allow MP to select the Boost version, so moving back to 1.75.0 or 1.71.0 or whatever is doable ... if it works with this SciPy of course :)

Reply:
I agree it's easy to allow MP to select the Boost versions, the question is how easy it is to tell SciPy to actually use that ;)

Reply:
Pretty easy it turns out ...

Reply:
works with Boost 1.71.0 ... so we're good to go without having to create more Boost ports